controllers/booksController.js

```diff
@@ -1,15 +1,17 @@
 import Book, { sortBooks } from '../model/Book.js';
 
 export const handleNewBook = async (req, res) => {
+  // TODO add meta data fields
   const { title, author, readOn } = req.body;
   if (!title) {
     return res.status(400).json({ 'message': 'Title is required' });
   }
 
-  const duplicate = await Book.findOne({ title: title }).exec();
+  const match = new RegExp(title, 'i');
+  const duplicate = await Book.findOne({ title: { $regex: match }}).exec();
```

Comment:
you can query by regex in MongoDB and that seemed like the simplest way to do a case insensitive look up for title. Might throw the author in at some point but NOT in this PR